src/test/java/com/conveyal/gtfs/dto/BookingRuleDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.conveyal.gtfs.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.net.URL;
+
+/**
+ * DTO used to model expected {@link com.conveyal.gtfs.model.BookingRule} JSON structure for the editor. NOTE: reference types
+ * (e.g., Integer and Double) are used here in order to model null/empty values in JSON object.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BookingRuleDTO {
+    public int id;
+    public String booking_rule_id;
+    public int booking_type;
+    public int prior_notice_duration_min;
+    public int prior_notice_duration_max;
+    public int prior_notice_last_day;
+    public int prior_notice_last_time;
```

Comment:
Should `prior_notice_start|end_time` be String or Time?

Reply:
Same issue as above. I believe other times are ints which is why this one is too

Reply:
Not here. They are defined as `TimeField` in `Table.java`, but here they should be ints matching other DTOs.